lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, ViewEncapsulation, EventEmitter, Output } from "@angular/core";
+
+@Component({
+    selector: "sebng-chip",
+    styleUrls: ["./chip.component.scss"],
+    templateUrl: "./chip.component.html",
+    encapsulation: ViewEncapsulation.None,
```

Comment:
we don't do this anymore, just omit encapsulation altogether

Reply:
Why though ?

Reply:
Not providing one means it takes the default being `Emulated` | `None` . I believe there was a reason why we agreed on the none long ago. When was this changed ?

Reply:
The issues we faced then were due to the way angular was encapsulating before. I don't think this issue is there anymore. @omarVodiaK do you remeber?

Reply:
> The issues we faced then were due to the way angular was encapsulating before. I don't think this issue is there anymore. @omarVodiaK do you remeber?

I remember the encapsulation issue and we arrived at that after realising how it messes with our components and styling encapsulations. My question is how was this issue resolved ?

Reply:
Ok forget about why we used it before. 

https://dev.to/monicafidalgo/what-is-viewencapsulation-in-angular-470o

ViewEncapsulation.Emulated (DEFAULT)
The Emulated mode is the default one. This allows that styles from main HTML propagate to the component but styles defined in this component's @Component decorator are scoped to this component only.

ViewEncapsulation.None
In the None mode, styles from the component propagate back to the main HTML and therefore are visible to all components on the page. Be careful with apps that have None components in the application.

Which one do you think makes more sense to use?

Reply:
The `None` version makes more sense. We are trying to make as close as possible to an HTML only layout. The emulated version will change class names and add weird stuff just to implement encapsulation for those developers who don't know how to achieve it with CSS, but we do know how to do it, so there is no need for it.

Reply:
I am aware of the Encapsulations types and thanks to the issues we had long ago, we decided to use the `None` for our component. We had a reason back then and my question is has the reason changed and why ?

Reply:
If you read what I posted you will see that None enables your component CSS to propagate back to the main html which can cause bugs. 

Do you remember what the issue was originally and why the decided to use None? I prefer to go with default angular behaviour whenever possible.